x-pack/plugins/security_solution/server/endpoint/lib/artifacts/task.ts

```diff
@@ -105,22 +109,34 @@ export class ManifestTask {
     // Check that this task is current
     if (taskId !== this.getTaskId()) {
       // old task, return
-      this.logger.debug(`Outdated task running: ${taskId}`);
+      this.logger.info(`Outdated task running: ${taskId}`);
       return;
     }
 
     const manifestManager = this.endpointAppContext.service.getManifestManager();
 
     if (manifestManager === undefined) {
-      this.logger.debug('Manifest Manager not available.');
+      this.logger.error('Manifest Manager not available.');
```

Comment:
This is another one I felt was important to know because TaskManger should always be available